include/swift/AST/DiagnosticsParse.def

```diff
@@ -1789,6 +1795,9 @@ ERROR(avail_query_version_comparison_not_needed,
 ERROR(availability_query_wildcard_required, none,
       "must handle potential future platforms with '*'", ())
 
+ERROR(unavailability_query_wildcard_not_required, none,
+      "platform wildcard '*' is implicit in unavailability", ())
```

Comment:
I think it would be slightly better to rephrase this to something like:

```
platform wildcard '*' is always implicit in '#unavailable'
```

The "always" indicates that it is (erm) always implicit, never explicit. Also "unavailability" is a mouthful of a word (it's 7 syllables), other diagnostics are using `'#unavailable'` (5 syllables) so IMO it makes sense to be consistent.